src/Roslyn.Diagnostics.Analyzers/UnitTests/DeclarePublicAPIAnalyzerTests.cs

```diff
@@ -532,6 +532,28 @@ public class C
                 GetAdditionalFileResultAt(7, 1, shippedFilePath, DeclarePublicAPIAnalyzer.RemoveDeletedApiRule, "C.Method() -> void"));
         }
 
+
+        [Fact]
+        public void TypeForwardsAreProcessed()
+        {
+            var source = @"
+[assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.StringComparison))]
+";
+
+            string shippedText = $@"
+System.StringComparison
```

Comment:
:question: Should we start these lines with `[TypeForwardedTo]` (or something like that)? If in the future a change is made to the forwarded type in the target assembly, it would show up as a change to the public API in the source assembly, which could be misleading during an API review if it's not clear that the change occurred due to forwarding.